### PR TITLE
Setup travis ci

### LINKED ISSUE #6: set up Travis CI

### PR DESCRIPTION
**Pull request by skizza8**
## Description
This is a  configuration file (travis.yml) that will run the build commands for our CI/CD pipeline with Travis-CI.

Fixes #6 

## How Has This Been Tested?
This has been tested on the Travis-CI platform.
https://travis-ci.org/github/BuildForSDG/Teachers-Hub-Backend

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "3.8"
+#create requirements.txt file
+before_install:
+  - pip freeze --local > requirements.txt
```
Comment (mariamiah) [inline .travis.yml]: can we get rid of this line along with the before_install, since we already have a requirements.txt file in the root

Comment (skizza8) [reply .travis.yml]: Done.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  # First sample run test on PIP version
+  - pytest tests/test_src.py
+notifications:
+  email:
+    - skizza8@gmail.com
```
Comment (mariamiah) [inline .travis.yml]: can we remove the notifications bit, you wouldn't want to receive a notification about each and every build

Comment (skizza8) [reply .travis.yml]: Done.

**Review by mariamiah — APPROVED** [review]
LGTM!

**Review by AineKiraboMbabazi — APPROVED** [review]
This looks good for now

**Pull request merged**